metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/__init__.py

```diff
@@ -1,6 +1,6 @@
 # Published at https://pypi.org/project/acryl-datahub/.
 __package_name__ = "acryl-datahub-airflow-plugin"
-__version__ = "0.10.1+affirm4"
+__version__ = "0.10.1+affirm5"
```

Comment:
Is there a point in bumping up these versions when there is really no change? Maybe we can just replace affirm4 on Artifactory?

Reply:
I noticed that when we don't bump up the version, make docker_image is not downloading the latest binary.

Reply:
Why do we need a docker image? This should just be the python package on Artifactory?

Reply:
Eventually packages have to be backed into a docker image. During docker image build all the Python packages will be downloaded from artifactory.